### DISCUSSION TITLE
How to change titles in resume.json? for ex. from "work" to "experience"

### DISCUSSION THREAD

**Discussion by cvbate** [alshedivat/al-folio#2274] [Q&A] [answered]
Hello, I am trying to change the names of the arrays for each section of the resume.json. However when I change the name in the json file, it removes the whole section from my site! I figured that there must be some variable defined elsewhere that calls these arrays by the name listed in the .json that'll have to change but I cant figure it out. I checked cv.liquid under _layouts but i didn't find anything(but maybe that's just me). I know I could just use the cv.md but I really want to get the .json to work! thank you! 

EDIT: So i actually found it in cv.liquid after looking(again) and I was able to change the names of my arrays, however they are still not showing up on my site! See image below to see where i changed "volunteer" to "research". I copied and pasted it to the name of the array in resume.json so I know they are identical....but still nothing!
![image](https://github.com/alshedivat/al-folio/assets/122933733/78091fa4-3792-495a-bedf-5abc5e4da082)

**Comment by george-gca** [comment]
TLDR: you are changing in the wrong part of the code.

First of all there is a precedence of CV source, as stated [in the docs](https://github.com/alshedivat/al-folio/blob/master/CUSTOMIZE.md#modifying-the-cv-information). Based on what is written there, the order is:
1. resume data defined in [\_config.yml](https://github.com/alshedivat/al-folio/blob/master/_config.yml) and loaded via a json file
2. [\_data/cv.yml](https://github.com/alshedivat/al-folio/blob/master/_data/cv.yml) if the first was not found

So this order is implemented there in [\_layouts/cv.liquid](https://github.com/alshedivat/al-folio/blob/master/_layouts/cv.liquid). `{% unless site.data.resume %}` means that will do what is inside this block **unless** `site.data.resume` doesn't exist. If it doesn't, it will do what is inside the `{% else %}` block.

So you are changing the code that handles the second case, not the first, which is your case.

**Reply by cvbate** [reply]
Okay, thank you for your response! Do you (or anyone else) have a suggestion to change the titles? I looked in the config.yml and i saw this:
![image](https://github.com/alshedivat/al-folio/assets/122933733/831354b7-2bf0-49f4-aa36-5868d6240ed3)

however changthing these to match what I changed in the jsonfile doesn't work either- nor does just changing the name in the json file itself.

**Reply by abhilesh** [reply, marked as answer]
I believe this limitation is due to the schema used in [JSON Resume](https://jsonresume.org/schema), which wouldn't recognize "experience" as a valid field. Hence, the titles don't change when you modify`resume.json` (or anything else for that matter).

To make modifications to the titles, you'd have to revert to using the `_data/cv.yml` file. [For this to be triggered](https://github.com/alshedivat/al-folio/blob/master/CUSTOMIZE.md#modifying-the-cv-information:~:text=What%20this%20means%20is%2C%20if%20there%20is%20no%20resume%20data%20defined%20in%20_config.yml%20and%20loaded%20via%20a%20json%20file%2C%20it%20will%20load%20the%20contents%20of%20_data/cv.yml.%20If%20you%20want%20to%20use%20the%20_data/cv.yml%20file%20as%20the%20source%20of%20your%20CV%2C%20you%20must%20delete%20the%20assets/json/resume.json%20file.), you must delete the `assets/json/resume.json` file.

Then, modify your `_data/cv.yml` file, where you can change the `title: ` field to reflect the desired title.
```
- title: General Information
  type: map
  contents:
    - name: Full Name
      value: Albert Einstein
    - name: Date of Birth
      value: 14th March 1879
    - name: Languages
      value: English, German
```

Changing `title: General Information` to `title: Basics` would change the header of the first entry in your resume to `Basics`
